firefox_ui_harness/runtests.py

```diff
@@ -14,10 +15,18 @@
 
 
 class ReleaseTestRunner(BaseMarionetteTestRunner):
+    extra_prefs = {
+        "browser.tabs.remote.autostart": True,
+    }
 
     def __init__(self, *args, **kwargs):
         if not kwargs.get('server_root'):
             kwargs['server_root'] = firefox_ui_tests.resources
+        prefs = kwargs.get('prefs', {})
+        extra_prefs = copy.deepcopy(ReleaseTestRunner.extra_prefs)
+        extra_prefs.update(prefs)
+        kwargs['prefs'] = extra_prefs
```

Comment:
This is just so we don't modify the class variable unnecessarily. Either way is probably ok in this case.